godtools/App/Share/Data/TranslationsRepository/ManifestParser/TranslationManifestParser.swift

```diff
@@ -39,32 +40,57 @@ class TranslationManifestParser {
         self.resourcesFileCache = resourcesFileCache
     }
     
-    func parse(manifestName: String) -> Result<Manifest, Error> {
+    func parsePublisher(manifestName: String) -> AnyPublisher<Manifest, Error> {
+        
+        return Future() { promise in
+
+            self.parseAsync(manifestName: manifestName) { (result: Result<Manifest, Error>) in
                 
+                switch result {
+                    
+                case .success(let manifest):
+                    promise(.success(manifest))
+                    
+                case .failure(let error):
+                    promise(.failure(error))
+                }
+            }
+        }
+        .eraseToAnyPublisher()
+    }
+    
+    private func parseAsync(manifestName: String, completion: @escaping ((_ result: Result<Manifest, Error>) -> Void)) {
+        
         let location: FileCacheLocation = FileCacheLocation(relativeUrlString: manifestName)
         
         switch resourcesFileCache.getFileExists(location: location) {
         
         case .success(let fileExists):
             
             guard fileExists else {
-                return .failure(NSError.errorWithDescription(description: "Could not find translation manifest file in file cache."))
+                completion(.failure(NSError.errorWithDescription(description: "Could not find translation manifest file in file cache.")))
+                return
             }
             
         case .failure(let error):
-            return .failure(error)
+            completion(.failure(error))
+            return
         }
         
-        let result: ParserResult = self.parser.parseManifestBlocking(fileName: manifestName, config: self.parserConfig)
```

Comment:
guess you weren't using the `parseManifestBlocking(fileName: manifestName)` variant that didn't require passing a config object.